### PR TITLE
Structure resource layer - Wave 1️⃣ 

### PR DESCRIPTION
**Pull request by flvndvd**
## Description

This PR initiates a reorganization of the connectors project, introducing a structured `resources` folder. This new organizational schema is designed with the following architecture:

- The `resources` folder serves as the core for all domain-business logic.
- Within it, a `storage` subfolder consolidates all database-related logic, ensuring a centralized approach to our database interactions.
- Further nested, the `schemas` subfolder hosts all Sequelize models and, soon, Drizzle schemas, ensuring a cohesive storage of our data structure definitions.

The rationale behind consolidating these elements into a unified `resources` folder is:
- Aims to maintain a clear and organized codebase by grouping related files, to ease navigation and project coherence.
- Positioning schema definitions in proximity to their corresponding resources foster a close association between data structures and their operational logic.
- This restructured approach will help scale as needed.

## Scope

This PR simply moves the Sequelize connection to the new structure and refactor all call sites. In the longer term, this connection will never be exposed outside of the `resources` folder.

## Deploy Plan

There is no migration to run. The CI flags those changes as risky but it's purely naming.

### CODE REVIEW

File: connectors/src/lib/models/index.ts [hunk 1]
```diff
@@ -8,16 +8,9 @@ import type {
   InferAttributes,
   InferCreationAttributes,
 } from "sequelize";
-import { DataTypes, Model, Sequelize } from "sequelize";
+import { DataTypes, Model } from "sequelize";
 
-const { CONNECTORS_DATABASE_URI } = process.env;
```
Comment (flvndvd) [inline connectors/src/lib/models/index.ts:13]: Main change.

**Review by fontanierh — APPROVED** [review]
Looks great ! 

Wondering about "storage" as a name for the DB connection stuff. Why not just "database" ?

**Comment by flvndvd** [comment]
> Looks great !
> 
> Wondering about "storage" as a name for the DB connection stuff. Why not just "database" ?

The goal is to think longer term, as we could end up saving stuff in a DFS, use some caching as well.

**Review by spolu — APPROVED** [review]
👍

**Pull request merged**